ydb/docs/en/core/yql/reference/yql-core/udf/list/knn.md

```diff
@@ -31,6 +50,30 @@ In {{ ydb-short-name }}, calculations are performed on the `String` data type, w
 
 Vector functions are implemented as user-defined functions (UDF) in the `Knn` module.
 
+### Functions for converting between vector and binary representations
+
+Conversion functions are needed to serialize the vector set into an internal binary representation and vice versa.
+
+All conversation functions returns [Tagged](../../types/special.md) types.
+
+The binary representation of a vector can be persisted in a {{ ydb-short-name }} table column, {{ ydb-short-name }} doesn't support storing `Tagged` types, so user should store them as `String` type using [Untag](../../builtins/basic#as-tagged) function.
```

Comment:
```suggestion
{% note info %}

To persist vector's binary representation in a {{ ydb-short-name }} table column,  use [Untag](../../builtins/basic#as-tagged) function first because {{ ydb-short-name }} doesn't support storing `Tagged` types.

{% endnote %}
```